src/Controls/src/Core/TabbedPage/TabbedPage.Windows.cs

```diff
@@ -19,7 +19,7 @@ public partial class TabbedPage
 		WFrame? _navigationFrame;
 		bool _connectedToHandler;
 		WFrame NavigationFrame => _navigationFrame ?? throw new ArgumentNullException(nameof(NavigationFrame));
-		IMauiContext MauiContext => this.Handler?.MauiContext ?? throw new InvalidOperationException("MauiContext cannot be null here");
+		IMauiContext? MauiContext => this.Handler?.MauiContext;
```

Comment:
I think we want to leave the exception

Can you just add a method to check nullability? 

bool HasHandler => this.Handler?.MauiContext is not null

Reply:
@PureWeen MauiContext reference is used in multiple places in this file, and I have ensured that null checks are added to prevent exceptions. Do we need to assign this.Handler?.MauiContext based on HasHandler in each place?

Reply:
I think we need to take a different approach here. 

we should be no longer listening to the SelectionChanged event after the handler is disconnected but it looks like we are

![image](https://github.com/user-attachments/assets/274b0935-8041-4cd5-a2a6-481f712c2d61)

Can we try to figure out here why the subscription still exists after disconnect is called?

Reply:
@PureWeen Upon checking further, the SelectionChanged event was unhooked properly during navigation, but the _navigationFrame.Navigated event was not. Since I am unable to reproduce the exact crash, based on the call stack from the dump files, I suspect that the issue occurs because the Navigated event was not unhooked properly. I have now unhooked the Navigated event correctly. If the Navigated event is properly unhooked during the disconnect handling, it will prevent the UpdateCurrentPageContent method from being called, even when triggered from the selection changing event. Could you please review and share your concerns?